recipes/hepmc2/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Renaming this to `run_test.sh` will cause it to be ran implicitly (or `run_test.py`/`run_test.bat`)

Reply:
The error is reported as 
line 7 command not found

Reply:
That makes sense as it's not in `PATH`, running it explicitly would need `${RECIPE_DIR}/test.sh`